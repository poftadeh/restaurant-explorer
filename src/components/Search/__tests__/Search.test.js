import React from 'react';
import { shallow } from 'enzyme';
import { Search } from '../index';

it('renders <Search /> properly', () => {
  const wrapper = shallow(<Search />);
  expect(wrapper).toMatchSnapshot();
});
