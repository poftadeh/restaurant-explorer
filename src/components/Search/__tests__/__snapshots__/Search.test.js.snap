// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Search /> properly 1`] = `
<div
  className="search-container"
>
  <label
    htmlFor="city"
  >
    <span
      className="label-text"
    >
      City
    </span>
    <input
      className="search-input"
      id="city"
      onChange={[Function]}
      type="text"
    />
  </label>
  <label
    htmlFor="refine-text"
  >
    <span
      className="label-text"
    >
      Refine
    </span>
    <input
      className="search-input"
      id="refine-text"
      onChange={[Function]}
      type="text"
    />
  </label>
</div>
`;
